### PR TITLE
Add pony orm

### PR DESCRIPTION
**Pull request by dharhas**

### CODE REVIEW

File: recipes/pony/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+test:
+  imports:
+    - pony
+    - pony.orm
+    - pony.utils
+
+about:
+  home: https://ponyorm.com
+  license: Apache-2.0
+  license_family: Apache
```
Comment (scopatz) [inline recipes/pony/meta.yaml]: Hi @dharhas, I am pretty sure the Apache 2 license, requires the `license_file` entry. If the license isn't in the pypi tarball, you can package it manually in the recipe: http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

Comment (dharhas) [inline recipes/pony/meta.yaml]: thanks for the link. the Licence wasn't in the pypi tarball. I'll package it.

**Comment by dharhas** [comment]
So appveyor seems to have failed on the conda update. How do I restart the the build?

**Comment by ocefpaf** [comment]
Just did that for you.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**